Makefile

```diff
@@ -26,12 +27,51 @@ start-containers: clean-containers
 	@echo "  >  Starting containers"
 	@cd $(DOCKER) && docker-compose up -d
 
-# Backward compatibility
+# Alias for integration-test
 testall: integration-test
 
-# Setup pretest as a prerequisite of tests.
+
+## Integration test with Kafka and Zookeper
 integration-test: pretestinfra
 
 pretestinfra:
 	@echo Setting up Zookeeper and Kafka. Docker required.
 	@$(MAKE) start-containers
+
+.DEFAULT:
```

Comment:
What's the difference between `.DEFAULT` and `.DEFAULT_GOAL`?

Reply:
.DEFAULT is executed when you pass a target but there is no match. Example: make dummy

.DEFAULT_GOAL is executed when you do not provide a target.